repository/gogit_test.go

```diff
@@ -66,3 +66,36 @@ func TestNewGoGitRepo(t *testing.T) {
 func TestGoGitRepo(t *testing.T) {
 	RepoTest(t, CreateGoGitTestRepo, CleanupTestRepos)
 }
+
+func TestGoGitRepo_Indexes(t *testing.T) {
+	t.Parallel()
+
+	plainRoot, err := ioutil.TempDir("", "")
+	require.NoError(t, err)
+	// defer os.RemoveAll(plainRoot)
```

Comment:
```suggestion
	defer os.RemoveAll(plainRoot)
```

I think this should be enabled.